src/Microsoft.InsightsGenerator/RulesEngine.cs

```diff
@@ -3,81 +3,200 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Text;
 using System.Text.RegularExpressions;
 
 namespace Microsoft.InsightsGenerator
 {
     public class RulesEngine
     {
-        public class ColumnHeaders
+        public static List<Template> Templates;
+
+        public static List<string> TopListHashHeaders = new List<string>{ "#top", "#average", "#averageSlice", "#topPerslice" , "#bottom"};
```

Comment:
not used?

Reply:
It's used to check if the text replacement needs extra step!